book/website/community-handbook/consistency/consistency-formatting.md

```diff
@@ -0,0 +1,194 @@
+(ch-consistency-formatting)=
+# Formatting
+
+Formatting refers to how _The Turing Way_ book is written and directly affects the book's appearance and presentation.
+
+_The Turing Way_ is hosted online as a Jupyter Book and uses the formatting recommendations as described in their [documentation](https://jupyterbook.org/intro.html).
+Proper formatting ensures that _The Turing Way_ is readable, accessible, and resembles a modular piece of work.
+
+
+(ch-consistency-formatting-hr)=
+## Hard Requirements
+
+The hard requirements in the consistency checklist ensure that _The Turing Way_ prioritises accessibility, collaboration, readability and ease of use. 
```

Comment:
Maybe good to mention (again) that the hard requirements are 'necessary' and the soft ones are 'nice-to-have'?